docs/api/upgrade/function.md

```diff
@@ -0,0 +1,22 @@
+# Function
+
+- [NgMetadataUpgradeAdapter](#NgMetadataUpgradeAdapter)
+
+---
+
+## NgMetadataUpgradeAdapter
```

Comment:
I would also describe all method available on instance of `NgMetadataUpgradeAdapter`. user will see them immediately thanks to TS, but I would like to have them in docs to be consistent